### ISSUE TITLE
Collect more metrics for perf runs

### ISSUE THREAD

**Issue by poornimag** [sac-urs/glusterfs-perf#26]
For each perf run, along with iops, latency etc. it's good to collect the following if possible:
- profile info
- network bandwidth usage
- disk usage
- CPU consumption of Gluster processes
- memory consumption of Gluster processes

**Comment by xhernandez** [comment]
I would add statedumps with latency enabled, both on bricks and clients. This gives us a bit more information about which xlator is actually contributing to the latency and how much.

**Comment by amarts** [comment]
Good thing is, latest codebase, (after 4.1), this is enabled by default?

**Comment by xhernandez** [comment]
> Good thing is, latest codebase, (after 4.1), this is enabled by default?

Yes, it seems that latency dump is enabled by default from v4.0.0. This version also incorporates the dump of monitoring metrics. Not sure if they are widely used, but probably it won't hurt to also have them.
